parser/parser.go

```diff
@@ -1789,6 +1824,16 @@ func isLiteralType(x ast.Expr) bool {
 	return true
 }
 
+// isRangeElem determine whether it is the type required by the element
+func isRangeElem(x ast.Expr) bool {
```

Comment:
not allow expr? eg. `1+2:10<<1:2`

Reply:
sorry, I added it when I debugged the problem, but I forgot to delete it